dist/index.js

```diff
@@ -8719,11 +8750,12 @@ function createBuild(params) {
         if (!buildConfig) {
             buildConfig = 'production';
         }
-        if (!baseHref) {
-            baseHref = '/';
+        let baseHrefString = '';
+        if (baseHref && baseHref.length > 0) {
+            baseHrefString = '--base-href=' + baseHref;
```

Comment:
Im basically just moving the appending into this if statement instead of at the end